packages/mdc-textfield/_mixins.scss

```diff
@@ -416,26 +367,14 @@
 @mixin mdc-text-field-fullwidth_ {
   width: 100%;
 
-  .mdc-text-field__input {
-    resize: vertical;
-  }
-
   &:not(.mdc-text-field--textarea) {
+    @include mdc-states-base-color(transparent);
```

Comment:
I'm seeing states overridden to transparent several places in this file. I would sooner add `:not` selectors to our application of states styles than override it everywhere else (which will create far more styles in the output CSS; try `npm run build` and see). Is there a reason you didn't do that?

Reply:
i think per our conversation its better to not have them too specific for the user to be able to customize it?